translations/messages.fr.yml

```diff
@@ -1028,6 +1028,10 @@ admin.thematic_community_membership.type.adherent: Adhérent
 admin.thematic_community_membership.type.electedRepresentative: Élu
 admin.thematic_community_membership.type.contact: Contact
 
+thematic_community_chief.popup.title: Charte de bonne utilisation des outils numériques
+thematic_community_chief.popup.subtitle: à l’usage des Responsables Communautés Thématiques
+thematic_community_chief.popup.file_url: https://storage.googleapis.com/en-marche-fr/TECH/charte_numerique_responsables_ct.pdf
```

Comment:
I need someone to create this file for me :)

Reply:
done

Reply:
thanks